vertical-pod-autoscaler/pkg/updater/main.go

```diff
@@ -53,6 +53,9 @@ var (
 	evictionRateBurst = flag.Int("eviction-rate-burst", 1, `Burst of pods that can be evicted.`)
 
 	address = flag.String("address", ":8943", "The address to expose Prometheus metrics.")
+
+	useAdmissionControllerStatus = flag.Bool("use-admission-controller-status", true,
+		"If Admission Contoller status should be used for making eviction decisions")
```

Comment:
More descriptive would be good:
If true, updater will only evict pods when admission controller status is valid.

Reply:
Done.